.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build
+on:
+  push:
+    branches:
+      main
+  pull_request:
+    types: [opened, synchronize, reopened]
+    
+jobs:
+  Run_Tests_Build_Artifact:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Static Code Analysis
+        run: mvn -B compile -Pmaas-static-code-analysis --file service/pom.xml
+      - name: Unit Tests
+        run: mvn -B test -Dcheckstyle.skip -Dpmd.skip -Dcpd.skip -Dfindbugs.skip -Dspotbugs.skip --file service/pom.xml
```

Comment:
we do not want to re-run PMD/findbugs test again in this step, since they have been executed in previous step.

Reply:
we don't need to to save some cycles in the build but if i remember correctly the sca tools doesn't take a long time to execute.

Reply:
But I'm ok to remove the skip stages.